spec/models/submission_spec.rb

```diff
@@ -254,7 +254,7 @@
       end
 
       it 'return the count of user submissions found by user_email eq 1' do
-        expect(submission.count_submissions_of_user).to eq 1
+        expect(submission.count_submissions_of_user).to eq 2
```

Comment:
**nitpick:**

1.  The label of this test case should say "... eq 2"
2. The label of the test case above this should say "... when 1 is deleted" to differentiate them.